app/jobs/zendesk_job.rb

```diff
@@ -0,0 +1,35 @@
+# Zendesk Integrations
+class ZendeskJob
+  def self.bootstrap()
+    @@hostname = ENV['ZENDESK_HOSTNAME']
+    @@username = ENV['ZENDESK_USERNAME']
+    @@password = ENV['ZENDESK_PASSWORD']
+  end
+
+  def self.has_zendesk_integration()
+    return @@hostname && @@username && @@password
+  end
+
+  def self.create_ticket(description)
+    # TODO: async this action, currently takes almost 2s to finish request. Modal is open for too long, confusing users whether action occurrec.
```

Comment:
Agreed. I didn't think it'd take this long. Sounds like we may need to start using [Active Job](https://guides.rubyonrails.org/active_job_basics.html).

Reply:
Maybe a future feature?